### PR TITLE
add docs on how to create topics

### PR DESCRIPTION
**Pull request by aultimus**
* add docs on how to create topics
* add docs on how to connect to leader via another connection

in response to issue https://github.com/segmentio/kafka-go/issues/440

### CODE REVIEW

**Review by achille-roussel — APPROVED** [review]
Thanks for adding this example @aultimus !

File: README.md [hunk 1]
```diff
@@ -84,6 +84,50 @@ batch.Close()
 conn.Close()
 ```
 
+```go
+// to create topics
+topic := "my-topic"
+partition := 0
+
+conn, err := kafka.DialLeader(context.Background(), "tcp", "localhost:9092", topic, partition)
```
Comment (stevevls) [inline README.md:92]: For a new topic, will this call succeed or will it fail with something like `UnknownTopicOrPartition`?

**Review by stevevls — COMMENTED** [review]
We get a lot of bug reports where people are confused about the asynchronous nature of topic creation.  Would it make sense to add some docs or an example of waiting for the topic to exist?

File: README.md (hunk 1 above)
Comment (aultimus) [reply README.md:92]: I ran the code and calling DialLeader with an uncreated topic does not appear to yield an error

**Comment by aultimus** [comment]
> We get a lot of bug reports where people are confused about the asynchronous nature of topic creation. Would it make sense to add some docs or an example of waiting for the topic to exist?

This sounds like another good topics for docs, I hope to address this in a future pr

**Pull request merged**